llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -7388,13 +7401,32 @@ AArch64TargetLowering::LowerCall(CallLoweringInfo &CLI,
         ISD::INTRINSIC_VOID, DL, MVT::Other, Chain,
         DAG.getConstant(Intrinsic::aarch64_sme_set_tpidr2, DL, MVT::i32),
         TPIDR2ObjAddr);
+    OptimizationRemarkEmitter ORE(&MF.getFunction());
+    ORE.emit([&]() {
+      auto R = CLI.CB ? OptimizationRemarkAnalysis(DEBUG_TYPE, "SMELazySaveZA",
```

Comment:
nit: Rather than using `DEBUG_TYPE/aarch64-lower`, can we use `sme` instead? I'm not sure if there will be other places where we want to emit any SME-specific remarks, at which point the end-user can simply do `-Rpass-analysis=sme`.